### ISSUE TITLE
Does the refresh_ivm function refresh an incremental view manually and incrementally?

### ISSUE THREAD

**Issue by salilsaifuddin** [sraoss/pg_ivm#94]
I am trying to determine whether the `refresh_ivm` function refreshes an incremental view manually. If it does, I also need to know if the refresh is performed incrementally.

Could you please provide insights or examples that clarify:
1. Whether `refresh_ivm` can be used to manually refresh an incremental view.
2. If the manual refresh is indeed incremental.

Any help or guidance on this would be greatly appreciated!

@Jamal-B, @yugo-n , @tatsuo-ishii  , @hanefi

**Comment by yugo-n** [comment]
@salilsaifuddin 

> Whether refresh_ivm can be used to manually refresh an incremental view.

No. Currently, pg_ivm supports only immediate maintenance, that is, views can be incrementally updated  in automatic way only when a underlying table is modified. I would like to support deferred maintenance that enables to update a view incrementally by executing refresh_immv command manually in future, but it is not implemented yet.
